README.md

```diff
@@ -369,6 +369,98 @@ For each Playbook run request a message with the following format is sent to Clo
 
 See [foreman_rh_cloud](https://github.com/ShimShtein/foreman_rh_cloud) for details.
 
+## Onboarding guide
+
+If you are an application trying to integrate with playbook dispatcher, we hope you find the following information helpful.
+
+### Dispatching playbooks
+
+Currently, dispatching of playbooks is only possible with the [Internal/Private API](https://github.com/RedHatInsights/playbook-dispatcher/blob/master/schema/private.openapi.yaml), and it will only be available to accept requests within the cluster.
```

Comment:
I am wondering if we should move this to a separate file or is the `README` good enough for it?

Reply:
I think a separate file will be better

Reply:
Done! :+1: